SPECS/libarrow/libarrow.spec

```diff
@@ -0,0 +1,366 @@
+ 
+
+%build
+pushd cpp
+%cmake \
+  -DARROW_WITH_PROTOBUF=OFF \
+  -DARROW_FLIGHT:BOOL=OFF \
+  -DARROW_ORC=OFF \
+  -DARROW_PARQUET:BOOL=ON \
+  -DARROW_PYTHON:BOOL=ON \
+  -DARROW_JEMALLOC:BOOL=OFF \
+  -DARROW_SIMD_LEVEL=NONE \
+  -DARROW_RUNTIME_SIMD_LEVEL=NONE \
+  -DGRPC_SOURCE=SYSTEM \
+  -Dxsimd_SOURCE=SYSTEM \
+  -DARROW_WITH_BROTLI:BOOL=ON \
+  -DARROW_WITH_BZ2:BOOL=ON \
+  -DARROW_WITH_LZ4:BOOL=ON \
+  -DARROW_WITH_SNAPPY:BOOL=ON \
+  -DARROW_WITH_ZLIB:BOOL=ON \
+  -DARROW_WITH_ZSTD:BOOL=OFF \
+  -DARROW_USE_XSIMD:BOOL=ON \
+  -DARROW_BUILD_STATIC:BOOL=OFF \
+  -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+  -DCMAKE_COLOR_MAKEFILE:BOOL=OFF \
+  -DARROW_USE_CCACHE:BOOL=OFF \
+  -DCMAKE_UNITY_BUILD:BOOL=ON \
+  -DPARQUET_REQUIRE_ENCRYPTION:BOOL=ON \
+  -DPythonInterp_FIND_VERSION:BOOL=ON \
+  -DPythonInterp_FIND_VERSION_MAJOR=3 \
+
+ 
+export VERBOSE=1
+export GCC_COLORS=
+%cmake_build
+popd
+rm -rf /tmp/usr
+ 
+#--------------------------------------------------------------------
+%install
+  
+pushd cpp
+%cmake_install
+popd
+#--------------------------------------------------------------------
+
+%files
+%{_libdir}/libarrow.so.*
+
+%files doc
+%license LICENSE.txt
+%doc README.md NOTICE.txt
+%exclude %{_docdir}/arrow/
+ 
+%files devel
+%dir %{_includedir}/arrow/
+     %{_includedir}/arrow/*
+%exclude %{_includedir}/arrow/dataset/
+%if %{with use_flight}
+%exclude %{_includedir}/arrow/flight/
+%exclude %{_includedir}/arrow-flight-glib
+%endif
+%exclude %{_libdir}/cmake/Arrow/FindBrotliAlt.cmake
+%exclude %{_libdir}/cmake/Arrow/Findlz4Alt.cmake
+%exclude %{_libdir}/cmake/Arrow/FindORC.cmake
+%exclude %{_libdir}/cmake/Arrow/FindorcAlt.cmake
+%exclude %{_libdir}/cmake/Arrow/FindSnappyAlt.cmake
+%exclude %{_libdir}/cmake/Arrow/FindgRPCAlt.cmake
+%exclude %{_libdir}/cmake/Arrow/Findre2Alt.cmake
+%exclude %{_libdir}/cmake/Arrow/Findutf8proc.cmake
+%exclude %{_libdir}/cmake/Arrow/FindzstdAlt.cmake
+%exclude %{_libdir}/cmake/Arrow/FindThriftAlt.cmake
+%exclude %{_libdir}/cmake/Arrow/FindOpenSSLAlt.cmake
+%exclude %{_libdir}/cmake/Arrow/FindProtobufAlt.cmake
+%dir %{_libdir}/cmake/Arrow/
+     %{_libdir}/cmake/Arrow/ArrowConfig*.cmake
+     %{_libdir}/cmake/Arrow/ArrowOptions.cmake
+     %{_libdir}/cmake/Arrow/ArrowTargets*.cmake
+     %{_libdir}/cmake/Arrow/arrow-config.cmake
+%{_libdir}/libarrow.so
+%{_libdir}/pkgconfig/arrow-compute.pc
+%{_libdir}/pkgconfig/arrow-csv.pc
+%{_libdir}/pkgconfig/arrow-filesystem.pc
+%{_libdir}/pkgconfig/arrow-json.pc
+%{_libdir}/pkgconfig/arrow.pc
+%{_datadir}/arrow/gdb/gdb_arrow.py
+#%{_datadir}/gdb/auto-load/usr/lib64/libarrow.so.*-gdb.py
+
+
+%files -n parquet-libs
+%{_libdir}/libparquet.so.*
+
+%files -n parquet-libs-devel
+%dir %{_includedir}/parquet/
+     %{_includedir}/parquet/*
+%{_libdir}/cmake/Parquet/*.cmake
+%{_libdir}/libparquet.so
+%{_libdir}/pkgconfig/parquet*.pc
+ 
+%changelog
+* Fri Mar 22 2024 Himaja Kesari <himajakesari@microsoft.com> - 15.0.0-4
```

Comment:
Why not release 2? (one greater than the Fedora version)?

Reply:
added changelog till fedora release 3